.github/workflows/unused-deps.yml

```diff
@@ -0,0 +1,34 @@
+on:
+  workflow_dispatch: {}
+  # Once per day at 00:00 UTC
+  push:
+    branches: [main]
+  pull_request:
+
+env:
+  RUSTFLAGS: -D warnings
```

Comment:
Instead of having this, you can use `-- -D warnings` at the end `cargo udeps` line

Reply:
this doesn't work but moved it to step-specific env